src/audio/pipeline.c

```diff
@@ -616,6 +616,46 @@ static void pipeline_comp_trigger_sched_comp(struct pipeline *p,
 	list_item_append(&p->list, &ctx->pipelines);
 }
 
+/*
+ * Check whether pipeline is incapable of acquiring data for capture.
+ *
+ * If capture START/RELEASE trigger originated on dailess pipeline and reached
+ * inactive pipeline as it's source, then we indicate that it's blocked.
+ *
+ * @param rsrc - component from remote pipeline serving as source to relevant
+ *		 pipeline
+ * @param ctx - trigger walk context
+ * @param dir - trigger direction
+ */
+static inline bool pipeline_has_no_data(struct comp_dev *rsrc,
```

Comment:
[non blocking comment] Very generic name used for a very specific logic, i.e. is_capture_src_active() would tell what exactly this function verifies w/o need to look inside.

Reply:
Renamed to ultimately indicate what this function purpose is.